basyx.submodelrepository/basyx.submodelrepository-feature-mqtt/src/main/java/org/eclipse/digitaltwin/basyx/submodelrepository/feature/mqtt/MqttSubmodelRepository.java

```diff
@@ -126,6 +152,13 @@ public void deleteSubmodelElement(String submodelId, String idShortPath) throws
 		submodelElementDeleted(submodelElement, getName(), submodelId, idShortPath);
 	}
 
+	@Override
+	public void patchSubmodelElements(String submodelId, List<SubmodelElement> submodelElementList) {
+		decorated.patchSubmodelElements(submodelId, submodelElementList);
+		List<SubmodelElement> patchedSubmodelElements = decorated.getSubmodelElements(submodelId, PaginationInfo.NO_LIMIT).getResult();
```

Comment:
Is there any reason to use the NO_LIMIT request to get the patched submodel elements again? They are already being passed as parameter.